examples/thing/cert-registration/main.go

```diff
@@ -67,51 +44,59 @@ func loadKey() (crypto.Signer, error) {
 	return privateKey.(crypto.Signer), nil
 }
 
-func loadCertificates() ([]*x509.Certificate, error) {
-	var err error
-	var certBytes []byte
-	if *cert != "" {
-		certBytes = []byte(*cert)
-	} else {
-		certBytes, err = ioutil.ReadFile(*certFile)
-		if err != nil {
-			return nil, err
-		}
+func decodeCertificates(certs string) ([]*x509.Certificate, error) {
+	block, _ := pem.Decode([]byte(certs))
+	if block == nil {
+		return nil, fmt.Errorf("unable to decode certificate")
 	}
-	block, _ := pem.Decode(certBytes)
 	return x509.ParseCertificates(block.Bytes)
 }
 
 // certRegThing initialises a Thing with AM.
 // A successful initialisation means that the Thing has successfully registered and authenticated with AM.
-//
-// To create your own certificate. Open a bash terminal:
-// Choose an ID for your thing:
-//     thingName=thingOne
-// Use openssl to create a certificate signing request and a certificate:
-//     openssl req -new -sha256 -subj "/CN=${thingName}" -key ./examples/resources/eckey1.key.pem -out "${thingName}.csr.pem"
-//     openssl x509 -req -CA ./examples/resources/es256test.cert.pem -CAkey ./examples/resources/es256test.key.pem -CAcreateserial -in "${thingName}.csr.pem" -out "${thingName}.cert.pem"
-// Ensure that the ID and the path to the certificate is passed to this example:
-//     go run github.com/ForgeRock/iot-edge/v7/examples/thing/cert-registration -name "${thingName}" -certfile "${thingName}.cert.pem"
 func certRegThing() (err error) {
+	var (
```

Comment:
We will have to change the docs to use the new options. Can you create a story for this please?